packages/styles/src/layout/_containers.scss

```diff
@@ -7,26 +7,39 @@
 
 tokens.$default-map: components.$post-container;
 
-.container,
-.container-fluid {
-  width: 100%;
-  margin-inline: auto;
+@each $class in (container, container-fluid) {
```

Comment:
By adding this loop, you create two declaration blocks instead of one, increasing the size of the CSS output. 
Instead, you can leave the common styles in a block and create other blocks for styles that differs, like: 
```scss
.container,
container-fluid {
  // common styles
}

.container {
  // container offset and max-width  for the container
}

.container-fluid {
  // container offset for the fluid container
}
```